datastream-kafka-connector/src/test/java/com/linkedin/datastream/connectors/kafka/TestAbstractKafkaConnector.java

```diff
@@ -75,12 +77,72 @@ public void testConnectorRestartCalled() {
     connector.stop();
   }
 
+  @Test
+  public void testOnAssignmentChangeReassignment() {
```

Comment:
Will it be possible to add a test for the new metric you added to ensure it gets incremented when a task stop fails?

Reply:
I added a check for that in the test "testOnAssignmentChangeStopTaskFailure" below at line 133, I check for the metric count to be non-zero. Is that what you were suggesting?